.travis.yml

```diff
@@ -13,10 +13,17 @@ matrix:
     # Add builds for Conda.
     - python: 2.7
       env: CONDA=1
+      env: PIP=pip
```

Comment:
The override of `CONDA=1` here prevents a Conda build from running. That means we no longer produce Conda 2.7 packages and no longer run tests on that configuration.

We probably want a separate task for building wheels. That, or we integrate cibuildwheel into each individual job. However, the individual route is a bit more painful because cibuildwheel seems to insist on an opt-out list of platforms instead of an opt-in list. e.g. each individual job would need `CIBW_SKIP=cp27-* cp34-* cp35-*` or some such. That's a bit annoying. Probably easier to just produce the wheels in a separate job. But I worry about doing it that way because the wheels produced with cibuildwheel won't be the wheels used for tests. There is a potential for the build configuration to vary there. So I have to insist on using cibuildwheel in individual jobs and then running tests against the built wheel.

If you refactor this to not break Conda 2.7 builds, I can integrate cibuildwheel into other tasks on my own after it lands.

Reply:
Oops. Conda 2.7 builds have been re-enabled.